README.md

```diff
@@ -90,6 +86,41 @@ try {
 }
 ```
 
+
+### Using 2-legged OAuth2 using Client Credentials
+
+The Client Credentials grant is used when applications request an access token to access their own resources, not on behalf of a user.
+
+```php
+<?php
+
+// Bootup the Composer autoloader
+include __DIR__ . '/vendor/autoload.php';  
+
+use Mautic\Auth\ApiAuth;
+
+$settings = [
+    'AuthMethod'   => 'TwoLeggedOAuth2',
+    'clientKey'    => '',
+    'clientSecret' => '',
+    'baseUrl'      => '',
+];
+
+// $settings['accessToken'] = 'your stored access token';
+
+
+$initAuth = new ApiAuth();
+$auth     = $initAuth->newAuth($settings, $settings['AuthMethod']);
+
+if (!isset($settings['accessToken'])) {
+    // store it for one hour and use it in $settings above
```

Comment:
The token isn't always valid for one hour. Mautic users can set the access token lifetime under API settings:

![image](https://user-images.githubusercontent.com/17739158/135279925-c75605ff-d0c6-483e-99ae-bcebdeb7ddee.png)

... in which case `/oauth/v2/token` will return an `expires_in` of 7200 seconds (2 hrs) instead of 3600 seconds (1 hr):

```JSON
{
    "access_token": "TOKEN_HERE",
    "expires_in": 7200,
    "token_type": "bearer",
    "scope": null
}
```